test/e2e/specs/editor/blocks/table.spec.js

```diff
@@ -243,8 +243,11 @@ test.describe( 'Table', () => {
 		// Create the table.
 		await page.click( 'role=button[name="Create Table"i]' );
 
-		// Click the first cell and add some text.
-		await page.click( 'role=document[name="Block: Table"i] >> figcaption' );
+		await editor.clickBlockToolbarButton( 'Add caption' );
+		const caption = page.locator(
+			'role=textbox[name="Table caption text"i]'
+		);
+		await expect( caption ).toBeFocused();
```

Comment:
The purpose of this change is the same as for the gallery block.
Please see: https://github.com/WordPress/gutenberg/pull/47325#discussion_r1083260748